WcaOnRails/app/models/regional_organization.rb

```diff
@@ -5,8 +5,9 @@ class RegionalOrganization < ApplicationRecord
   has_one_attached :bylaws
   has_one_attached :extra_file
 
-  scope :currently_acknowledged, -> { where("end_date IS NULL OR end_date > ?", Date.today) }
-  scope :not_currently_acknowledged, -> { where("end_date < ?", Date.today) }
+  scope :currently_acknowledged, -> { where("start_date IS NOT NULL AND end_date IS NULL OR end_date > ?", Date.today) }
```

Comment:
I believe there should be parentheses, like so:
```sql
start_date IS NOT NULL AND (end_date IS NULL OR end_date > ?)
```
Right?

Reply:
Right! I'm updating this.